web/client/components/map/openlayers/Map.jsx

```diff
@@ -402,12 +402,22 @@ class OpenlayersMap extends React.Component {
         return !isEqual(resolutions, newResolutions);
     };
 
-    createView = (center, zoom, projection, options) => {
+    createView = (center, zoom, projection, options, oldProjection) => {
+        // reprojecting the extent of the newly created view. If the extent is set in the localConfig file
+        let newOptions = options;
+        if (options && options.extent) {
+            newOptions.extent = CoordinatesUtils.reprojectBbox(options.extent, oldProjection || 'EPSG:3857', projection);
```

Comment:
You are changing the original options object here, if you want to copy options in newOptions and then change it you should create newOptions this way:

const newOptions = assign({}, options);

More over, you always reproject, also if projection is not changed.

Reply:
you are mutating something that is in redux store, I believe (options are stored in layers configuration). This will lead to bugs: options are read-only here you cannot change them without the usual action -> reducer flow, so I would say you need to try a different place to do this transformation_:maybe the reducer that changes the projection?

Reply:
there is another approach witch is to transform the extent always from 'EPSG:3857',
the user must always set the extent in the localConfig using this crs coordinates ( it is the default anyway ).
what do you think ?